notebooks/EVA-release-RSIDquery.ipynb

```diff
@@ -0,0 +1,179 @@
15067307, 'version': 1, 'data': {'referenceSequenceAccession': 'GCA_000003055.5', 'taxonomyAccession': 9913, 'projectAccession': 'PRJEB6119', 'contig': 'GK000006.2', 'start': 87140970, 'referenceAllele': 'C', 'alternateAllele': 'T', 'clusteredVariantAccession': 379920406, 'supportedByEvidence': False, 'assemblyMatch': True, 'allelesMatch': True, 'validated': True, 'mapWeight': None, 'createdDate': '2014-12-11T06:43:00', 'remappedFrom': None, 'remappedDate': None, 'remappingId': None, 'backPropagatedVariantAccession': None}}, {'accession': 1829370055, 'version': 1, 'data': {'referenceSequenceAccession': 'GCA_002263795.2', 'taxonomyAccession': 9913, 'projectAccession': 'MARC_GPE_BULL_GENEX', 'contig': 'CM008173.2', 'start': 85411136, 'referenceAllele': 'C', 'alternateAllele': 'T', 'clusteredVariantAccession': 379920406, 'supportedByEvidence': True, 'assemblyMatch': True, 'allelesMatch': True, 'validated': False, 'mapWeight': None, 'createdDate': '2015-08-30T05:21:00', 'remappedFrom': 'GCA_000003055.5', 'remappedDate': '2021-08-15T21:20:26.241', 'remappingId': '778BEEA595920286BA9CC9ECF13442A69F9DA73B', 'backPropagatedVariantAccession': None}}, {'accession': 830915657, 'version': 1, 'data': {'referenceSequenceAccession': 'GCA_002263795.2', 'taxonomyAccession': 9913, 'projectAccession': 'IZ-PIB_SNP_INDEL_DISCOVERY', 'contig': 'CM008173.2', 'start': 85411136, 'referenceAllele': 'C', 'alternateAllele': 'T', 'clusteredVariantAccession': 379920406, 'supportedByEvidence': True, 'assemblyMatch': True, 'allelesMatch': True, 'validated': False, 'mapWeight': None, 'createdDate': '2013-08-05T22:40:00', 'remappedFrom': 'GCA_000003055.5', 'remappedDate': '2021-08-15T21:20:26.241', 'remappingId': '778BEEA595920286BA9CC9ECF13442A69F9DA73B', 'backPropagatedVariantAccession': None}}, {'accession': 1404048026, 'version': 1, 'data': {'referenceSequenceAccession': 'GCA_002263795.2', 'taxonomyAccession': 9913, 'projectAccession': 'HAFL_QUALITAS_20140601_SLIG', 'contig': 'CM008173.2', 'start': 85411136, 'referenceAllele': 'C', 'alternateAllele': 'T', 'clusteredVariantAccession': 379920406, 'supportedByEvidence': True, 'assemblyMatch': True, 'allelesMatch': True, 'validated': False, 'mapWeight': None, 'createdDate': '2014-10-18T00:49:00', 'remappedFrom': 'GCA_000003055.5', 'remappedDate': '2021-08-15T21:20:26.241', 'remappingId': '778BEEA595920286BA9CC9ECF13442A69F9DA73B', 'backPropagatedVariantAccession': None}}, {'accession': 1404048026, 'version': 1, 'data': {'referenceSequenceAccession': 'GCA_000003055.5', 'taxonomyAccession': 9913, 'projectAccession': 'HAFL_QUALITAS_20140601_SLIG', 'contig': 'GK000006.2', 'start': 87140970, 'referenceAllele': 'C', 'alternateAllele': 'T', 'clusteredVariantAccession': 379920406, 'supportedByEvidence': False, 'assemblyMatch': True, 'allelesMatch': True, 'validated': False, 'mapWeight': None, 'createdDate': '2014-10-18T00:49:00', 'remappedFrom': None, 'remappedDate': None, 'remappingId': None, 'backPropagatedVariantAccession': None}}]\n"
```

Comment:
If you want to show just an example, you can do `output[0]` (and then maybe print out `len(output)` to show how many SS you got).  On the other hand, if you're just demonstrating that you did get stuff back from the API (and a lot of stuff, hence parsing the JSON as you do next), then this is fine as it is.